README.md

```diff
@@ -97,6 +97,8 @@ The following environment variables are supported:
 | **CA_ALLOW_SUBJECT_ALT_NAMES**             | Whether or not SSL certificates containing Subject Alternative Names should be signed by the CA. Does nothing unless `CA_ENABLED=true`.<br><br>`false`        |
 | **PUPPET_REPORTS**                         | Sets `reports` in puppet.conf<br><br>`puppetdb`                                                                                                               |
 | **PUPPET_STORECONFIGS**                    | Sets `storeconfigs` in puppet.conf<br><br>`true`                                                                                                              |
+| **PUPPET_NODE_TERMINUS**                   | Sets `node_terminus` in puppet.conf<br><br>Defaults to unset.                                                                                                                                                                                                                                                                                                                                                               |
+| **PUPPET_EXTERNAL_NODES**                  | Sets `external_nodes` in puppet.conf<br><br>Defaults to unset.                                                                                                                                                                                                                                                                                                                                                              |
```

Comment:
where are you using these variables? 🤔

Reply:
D'oh, I forgot the setup script, sorry. Just added it.